ui/watch.bat

```diff
@@ -0,0 +1,33 @@
+rmdir %~dp0node_modules /s /q
```

Comment:
Please update the README and add a heading titled "Internationalization" that explains a bit about this, and anything else that would be relevant to set up or programmers new to the repo. Should be a couple paragraphs tops, and possibly just a sentence or two.